pkg/storage/replicate_queue.go

```diff
@@ -508,6 +508,7 @@ func (rq *replicateQueue) processOneChange(
 				); err != nil {
 					return false, err
 				}
+				rq.allocator.storePool.updateLocalStoreAfterRebalance(rebalanceReplica.StoreID, repl)
```

Comment:
We should just call this from within `addReplica` and `removeReplica` so that we always have more accurate estimates. That'll mean modifying `updateLocalStoreAfterRebalance` to support removing stats as well as adding them.

Reply:
@a-robinson  Do you mean we should add the `WritesPerSecond` when we `addReplica`, and we should minus the `WritesPerSecond` when we `removeReplica`?

Reply:
Yeah, that's what I was thinking.

Reply:
Done.